pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/ZkBasedTableRebalanceObserver.java

```diff
@@ -205,6 +209,17 @@ public int getNumUpdatesToZk() {
     return _numUpdatesToZk;
   }
 
+  /**
+   * Emits the rebalance progress in percent to the metrics. Uses the percentage of remaining segments to be added as
+   * the indicator of the overall progress.
+   * @param overallProgress the latest overall progress
+   */
+  private void emitProgressMetric(TableRebalanceProgressStats.RebalanceProgressStats overallProgress) {
+    _controllerMetrics.setValueOfTableGauge(_tableNameWithType + "." + _tableRebalanceContext.getOriginalJobId(),
```

Comment:
nit: let's add a comment to explain why original job ID is used instead of just the job ID?